drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -952,6 +943,16 @@ def _get_raw_earned_if_set(self):
         else:
             return None
 
+    def _get_weighted_earned_if_set(self):
+        """
+        Returns student's grade if already explicitly set, otherwise returns None,
+        with the problem weight applied.
+        """
```

Comment:
Nit. Syntax is a little baroque here.  Consider `"Returns student's grade with the problem weight applied if set, otherwise \`None\`"`.